vtkext/private/module/vtkF3DOpenGLGridMapper.cxx

```diff
@@ -89,8 +89,10 @@ void vtkF3DOpenGLGridMapper::ReplaceShaderValues(
   vtkShaderProgram::Substitute(FSSource, "//VTK::Color::Impl",
     "  float majorAlpha = antialias(min(majorGrid.x, majorGrid.y), gridLineWidth);\n"
     "  float minorAlpha = antialias(min(minorGrid.x, minorGrid.y), gridLineWidth);\n"
-    "  float zoomFadeFactor = 1.0 - clamp(fwidth(majorCoord.x / unitSquare * fadeDist), 0.0, 1.0);"
-    "  float alpha = max(majorAlpha, minorAlpha * minorOpacity * zoomFadeFactor);\n"
+    "  float zoomFadeFactor = 1.0 - clamp(fwidth(majorCoord.x / unitSquare * fadeDist), 0.0, 1.0);\n"
+    "  float axisThreshold = 0.005;\n"
```

Comment:
That fix looks correct to me.  
However, I think this threshold should be relative to the `unitSquare` value instead of an absolute value.  
What do you think @snoyer?

Reply:
idk, the alpha value would affect the width of the lines rather that how they are colored, wouldn't it? I think the issue is further down with the color mixing